<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QStackedWidget" name="setup_widgets_stack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background: red;</string>
        </property>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="styleSheet">
         <string notr="true">background: blue;</string>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="render_widgets_stack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_3">
        <property name="styleSheet">
         <string notr="true">background: red;</string>
        </property>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="styleSheet">
         <string notr="true">background: black;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <widget class="QMenu" name="menurecent">
     <property name="title">
      <string>Recent Projects</string>
     </property>
     <addaction name="actionfiles"/>
    </widget>
    <addaction name="action_new_project"/>
    <addaction name="action_load_project"/>
    <addaction name="menurecent"/>
    <addaction name="action_save_project"/>
    <addaction name="action_save_project_as"/>
    <addaction name="separator"/>
    <addaction name="action_import_geometry"/>
    <addaction name="action_import_piping"/>
    <addaction name="action_export_png"/>
    <addaction name="action_reset"/>
   </widget>
   <widget class="QMenu" name="menuGeneral_Settings">
    <property name="title">
     <string>Graphic</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGeneral_Settings_2">
    <property name="title">
     <string>General Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStructural_Settings">
    <property name="title">
     <string>Structural Model Setup</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAcoustic_Model_Setup">
    <property name="title">
     <string>Acoustic Model Setup</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModel_Info">
    <property name="title">
     <string>Model Info</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuResults_Viewer">
    <property name="title">
     <string>Results Viewer</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelo">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuGeneral_Settings"/>
   <addaction name="menuGeneral_Settings_2"/>
   <addaction name="menuStructural_Settings"/>
   <addaction name="menuAcoustic_Model_Setup"/>
   <addaction name="menuModel_Info"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuResults_Viewer"/>
   <addaction name="menuHelo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new_project"/>
   <addaction name="action_load_project"/>
   <addaction name="action_reset"/>
   <addaction name="action_export_png"/>
   <addaction name="separator"/>
   <addaction name="action_top_view"/>
   <addaction name="action_bottom_view"/>
   <addaction name="action_left_view"/>
   <addaction name="action_right_view"/>
   <addaction name="action_front_view"/>
   <addaction name="action_back_view"/>
   <addaction name="action_isometric_view"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_new_project">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/add.png</normaloff>../../../data/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="action_load_project">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/009-download.png</normaloff>../../../data/icons/009-download.png</iconset>
   </property>
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="action_export_png">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/save_image.png</normaloff>../../../data/icons/save_image.png</iconset>
   </property>
   <property name="text">
    <string>Export PNG</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/refresh.png</normaloff>../../../data/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_save_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_project_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_import_geometry">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import Geometry</string>
   </property>
  </action>
  <action name="action_import_piping">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import piping</string>
   </property>
   <property name="toolTip">
    <string>Import Piping</string>
   </property>
  </action>
  <action name="action_top_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/top.png</normaloff>../../../data/icons/views/top.png</iconset>
   </property>
   <property name="text">
    <string>Top View</string>
   </property>
  </action>
  <action name="action_bottom_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/bottom.png</normaloff>../../../data/icons/views/bottom.png</iconset>
   </property>
   <property name="text">
    <string>Bottom View</string>
   </property>
  </action>
  <action name="action_front_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/front.png</normaloff>../../../data/icons/views/front.png</iconset>
   </property>
   <property name="text">
    <string>Front View</string>
   </property>
  </action>
  <action name="action_back_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/back.png</normaloff>../../../data/icons/views/back.png</iconset>
   </property>
   <property name="text">
    <string>Back View</string>
   </property>
  </action>
  <action name="action_left_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/left.png</normaloff>../../../data/icons/views/left.png</iconset>
   </property>
   <property name="text">
    <string>Left View</string>
   </property>
  </action>
  <action name="action_right_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/right.png</normaloff>../../../data/icons/views/right.png</iconset>
   </property>
   <property name="text">
    <string>Right View</string>
   </property>
  </action>
  <action name="action_isometric_view">
   <property name="icon">
    <iconset>
     <normaloff>../../../data/icons/views/isometric.png</normaloff>../../../data/icons/views/isometric.png</iconset>
   </property>
   <property name="text">
    <string>Isometric View</string>
   </property>
  </action>
  <action name="actionfiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </action>
  <action name="action_geometry_workspace">
   <property name="text">
    <string>Geometry Workspace</string>
   </property>
  </action>
  <action name="action_mesh_workspace">
   <property name="text">
    <string>Mesh Workspace</string>
   </property>
  </action>
  <action name="action_results_workspace">
   <property name="text">
    <string>Results Workspace</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
